[2025-02-05T06:37:31.426+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: task_group_example.end_task manual__2025-02-05T06:36:32.485179+00:00 [queued]>
[2025-02-05T06:37:31.430+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: task_group_example.end_task manual__2025-02-05T06:36:32.485179+00:00 [queued]>
[2025-02-05T06:37:31.430+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2025-02-05T06:37:31.434+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): end_task> on 2025-02-05 06:36:32.485179+00:00
[2025-02-05T06:37:31.436+0000] {standard_task_runner.py:57} INFO - Started process 972 to run task
[2025-02-05T06:37:31.438+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'task_group_example', 'end_task', 'manual__2025-02-05T06:36:32.485179+00:00', '--job-id', '120', '--raw', '--subdir', 'DAGS_FOLDER/task_group_dag.py', '--cfg-path', '/tmp/tmpgphfk9ek']
[2025-02-05T06:37:31.438+0000] {standard_task_runner.py:85} INFO - Job 120: Subtask end_task
[2025-02-05T06:37:31.444+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-02-05T06:37:31.455+0000] {task_command.py:415} INFO - Running <TaskInstance: task_group_example.end_task manual__2025-02-05T06:36:32.485179+00:00 [running]> on host 3849bfb82152
[2025-02-05T06:37:31.480+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='task_group_example' AIRFLOW_CTX_TASK_ID='end_task' AIRFLOW_CTX_EXECUTION_DATE='2025-02-05T06:36:32.485179+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-05T06:36:32.485179+00:00'
[2025-02-05T06:37:31.481+0000] {logging_mixin.py:151} INFO - 执行任务: 结束任务
[2025-02-05T06:37:31.481+0000] {python.py:194} INFO - Done. Returned value was: None
[2025-02-05T06:37:31.483+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=task_group_example, task_id=end_task, execution_date=20250205T063632, start_date=20250205T063731, end_date=20250205T063731
[2025-02-05T06:37:31.496+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-02-05T06:37:31.502+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
